docs/_documentations/remote-configuring.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: docs
+title: Configuring Codewind remote
+description: Configuring your environment to use Codewind remote
+keywords: users, projects, Kubernetes, LDAP, user management, access management, login, deployment, pod, security, securing Cloud connection
+duration: 5 minutes
+permalink: remoteconfiguring
+type: document
+parent: installoncloud
+order: 1
+---
+
+## Configuring Codewind remote
+
+To securely configure Codewind remote, do the following steps.
+
+1. If you are using Kubernetes for Docker Desktop, validate you have an ingress controller installed on your cluster by typing the following command: [Mark to supply]
+
+   If you do not have an ingress controller, install one using the following commands:
+   - `kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/mandatory.yaml`
+   - `kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/provider/cloud-generic.yaml`
+
+   If you are developing on Windows [get map local address from Mark]
+
+   If you are on OpenShift, you must provide the ingress in the `cwctl install remote \` command during installation. 
+
+2. If you are using using Kubernetes for Docker Desktop, create the remote deployment by entering the following `cwctl` command: 
```

Comment:
Just noticed a typo. The word "using" appears twice. :)

Reply:
Typo resolved, could probably do with another review for such things Tuesday next week.